website/content/en/v0.28/troubleshooting.md

```diff
@@ -314,6 +314,35 @@ When a node is launched by Karpenter, it is assigned to a subnet within your VPC
 
 For more troubleshooting information on why your pod may have a `FailedCreateSandbox` error, view the [EKS CreatePodSandbox Knowledge Center Post](https://repost.aws/knowledge-center/eks-failed-create-pod-sandbox).
 
+#### Windows pods are failing to launch
+
+The following solution(s) may resolve your issue if you are seeing an error similar to the following when attempting to launch Windows pods.
+
+```
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx": plugin type="vpc-bridge" name="vpc" failed (add): failed to parse Kubernetes args: pod does not have label vpc.amazonaws.com/PrivateIPv4Address
+```
+
+This error typically occurs if you have not enabled Windows support. If you are provisioning Windows nodes, you need to [enable Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for your cluster by enabling Windows IPAM.
```

Comment:
The solution is repeated here. Just say here that `This error typically occurs if you have not enabled Windows support.`

Reply:
Got it. Will make the change.